docs/docsite/rst/community/create_pr_quick_start.rst

```diff
@@ -0,0 +1,269 @@
+
+2. Run sanity tests:
+
+  .. code-block:: bash
+
+    $ ansible-test sanity path/to/changed_file.py --docker -v
+
+  If they failed, look at the output carefully - it is informative and helps to identify a problem line quickly.
+  Sanity failings usually relate to incorrect code and documentation formatting.
+
+3. Run integration tests:
+
+  .. code-block:: bash
+
+    $ ansible-test integration name_of_test_subdirectory --docker -v
+
+  For example, if the tests files you changed are stored in ``tests/integration/targets/test_mysql_user/``, the command is:
+
+  .. code-block:: bash
+
+    $ ansible-test integration test_mysql_user --docker -v
+
+  You can use the ``-vv`` or ``-vvv`` argument if you need more detailed output.
+
+
+There are two possible outcomes:
+
+- They have failed. Look at the output of the command. Fix the problem place in the code and run again. Repeat the cycle until the tests pass.
+- They have passed. Remember they failed originally? Our congratulations! You have fixed the bug.
+
+In addition to the integration tests, you can also cover your changes with unit tests. This is often required when integration tests are not applicable to the collection.
+
+We use `pytest <https://docs.pytest.org/en/latest/>`_ as a testing framework.
+
+Files with unit tests are stored in the ``tests/unit/plugins/`` directory. If you want to run unit tests, say, for ``tests/unit/plugins/test_myclass.py``, the command is:
+
+.. code-block:: bash
+
+  $ ansible-test units tests/unit/plugins/test_myclass.py --docker
+
+If you want to run all unit tests available in the collection, run:
+
+.. code-block:: bash
+
+  $ ansible-test units --docker
+
+Submit a pull request
+=====================
+
+1. Commit your changes with an informative but short commit message:
+
+  .. code-block:: bash
+
+    $ git add /path/to/changed/file
+    $ git commit -m "module_name_you_fixed: fix crash when ..."
+
+2. Push the branch to the ``origin`` (your fork):
+
+  .. code-block:: bash
+
+    $ git push origin name_of_my_branch
+
+3. In a browser, navigate to the ``upstream`` repository (http://github.com/ansible-collections/COLLECTION_REPO).
+
+4. Click the :guilabel:`Pull requests` tab.
+
+  GitHub is tracking your fork, so it should see the new branch in it and automatically offer  to create a pull request. Sometimes GitHub does not do it, and you should click the :guilabel:`New pull request` button yourself. Then choose :guilabel:`compare across forks` under the :guilabel:`Compare changes` title.
+
+5.Choose your repository and the new branch you pushed in the right drop-down list. Confirm.
+
+  a. Fill out the pull request template with all information you want to mention.
+
+  b. Put ``Fixes + link to the issue`` in the pull request's description.
+
+  c. Put ``[WIP] + short description`` in the pull request's title. It's often a good idea to mention the name of the module/plugin you are modifying at the beginning of the description.
```

Comment:
Suggestion: delete "It's often a good idea to", and start the sentence with "Mention the name"